deployment/terraform/create.go

```diff
@@ -508,7 +514,7 @@ func (t *Terraform) setupLoadtestAgents(extAgent *ssh.ExtAgent, initData bool) e
 		return fmt.Errorf("error while setting up an agents: %w", err)
 	}
 
-	if !t.output.HasAppServers() {
+	if !t.output.HasAppServers() || !t.output.HasAgents() {
```

Comment:
`t.initLoadTest` already has this condition:

```go
if len(t.output.Agents) == 0 {
	return errors.New("there are no agents to initialize load-test")
}
```

Do we still need this OR condition?

Reply:
Right, my addition was to avoid the error. I don't think having zero agents is a scenario we should fail. But I'm 0/5 as in the end this is a load test tool :)

Reply:
Hmm, would there be any use-case where you want to deploy app servers, but not agents?

But even if we did, then in that case, we can remove the if condition inside `initLoadTest`.

But let's confirm whether an error is needed or not - WDYT @agarciamontoro ?

Reply:
This has been brought up before. I think it doesn't make a lot of sense right now, since we always want to run a load test, and for that we need agents. If we ended up splitting the tool in a deployer / loadtester pair, then that would make more sense. But I think we should always assume there's at least one agent. I'm happy to change my mind, but at least it should be in a separate PR, I feel it's a whole separate discussion.

Reply:
Let's change this @isacikgoz as per the above discussion.